pkg/health/health.go

```diff
@@ -28,24 +30,33 @@ type Result struct {
 	Output  string
 }
 
-func (h ConsulHealthChecker) Service(serviceID string) (map[string][]Result, error) {
-	entries, _, err := h.client.Health().Service(serviceID, "", false, nil)
+func (h ConsulHealthChecker) Service(serviceID string) (map[string][][]Result, error) {
```

Comment:
could we have a comment here to describe the characteristics of the `map[string][][]Result`? Kind of our fault for not documenting it before--I think even in its previous form `map[string][]Result` it could have deserved a comment about the structure (what will the keys be? given a key, what does the corresponding value represent?). But since you are changing it to have another array, I think it's a good time to doc.